### DISCUSSION TITLE
Horse menu Horse menu failure to inject causing crashes

### DISCUSSION THREAD

**Discussion by victordlh29** [YimMenu/HorseMenu#133] [Q&A] [answered]
I have a problem I inject the dll and the game gets stuck and does not respond.
[cout.log](https://github.com/YimMenu/HorseMenu/files/14964675/cout.log)

**Comment by Rxann** [comment]
This should be converted to an issue, more specifically a bug issue.

**Reply by victordlh29** [reply]
Can you help me with this error ?

**Comment by victordlh29** [comment, marked as answer]
I managed to fix it and it is working
